File: 0070-climbing-stairs/0070-climbing-stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        dp = [1, 1]
        i = 1
        while i <= n:
            temp = dp[1]
            dp[1] = dp[0] + dp[1]
            dp[0] = temp
            i = i + 1
        
        return dp[0]
        

File: 0070-climbing-stairs/0070-climbing-stairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
    let dp = [1, 1];

    for (let i = 0; i < n; i++) {
        let temp = dp[0];
        dp[0] += dp[1];
        dp[1] = temp;
    }

    return dp[1];
};